scripts/unassign-inactive-issues.mjs

```diff
@@ -0,0 +1,169 @@
+import { Octokit } from '@octokit/rest';
+import * as core from '@actions/core';
+import fetch from 'node-fetch';
+
+async function run() {
+  try {
+    const token = process.env.WEB_Token;
+    
+    if (!token) {
+      throw new Error('No authentication token provided. Please ensure WEB_Token is set in the workflow.');
+    }
+
+    console.log('Token exists:', !!token);
+
+    const inactivityPeriodInMinutes = 10;
+
+    const repository = process.env.GITHUB_REPOSITORY;
+    if (!repository) {
+      throw new Error('GITHUB_REPOSITORY environment variable is not set');
+    }
+
+    const [owner, repo] = repository.split('/');
+    console.log(`Processing repository: ${owner}/${repo}`);
+
+    const octokit = new Octokit({
+      auth: token,
+      request: {
+        fetch: fetch
+      }
+    });
+
+    try {
+      const authUser = await octokit.rest.users.getAuthenticated();
+      console.log('Successfully authenticated with GitHub as:', authUser.data.login);
+    } catch (authError) {
+      console.error('Authentication error details:', authError);
+      throw new Error(`Authentication failed: ${authError.message}. Please check your token permissions.`);
+    }
+
+    // List open issues
+    const issues = await octokit.rest.issues.listForRepo({
+      owner,
+      repo,
+      state: 'open',
+      per_page: 100,
+    });
+
+    console.log(`Found ${issues.data.length} open issues`);
+
+    for (const issue of issues.data) {
+      const assignee = issue.assignee;
+      if (!assignee || assignee.site_admin) {
```

Comment:
I think there could be more kinds of assignees that we'd need to skip, not just site admins. Is there a documentation of all the possibilities?

(even though not Octokit, here is one parallel where we're also excluding organization members, as not everybody is admin or owner  https://github.com/learningequality/.github/blob/main/.github/workflows/notify_team_new_comment.yml#L13-L18)

Reply:
I'll update the code by referencing the implementation in the provided link and ensure that it properly checks for both `OWNER` and `MEMBER` values in the author_association variable.

As for the possible values of author_association, here are the potential options:

1. `OWNER` - The user is an owner of the repository.
2. `MEMBER` - The user is a member of the repository.
3. `COLLABORATOR` - The user is a collaborator on the repository, typically with write access.
4. `CONTRIBUTOR` - The user has contributed to the repository but is not necessarily a member or collaborator.
5. `NONE` - The user has no direct association with the repository.

Reply:
Thank you for listing the options. Yes, let's exclude owners and members.